docs/source/building.rst

```diff
@@ -0,0 +1,112 @@
+.. _building:
+
+Building PYJNIus
```

Comment:
Perhaps this document could be much simpler:

We list the requirements for BUILDING for each platform, i.e.  JDK and
  - GCC for Linux
  - Visual Studio for Windows
  - Xcode tools for macOS

Then tell people that pip install will do the rest.

Reply:
Yeah, unlike `kivy/kivy` where we have a lot of dependencies and things to build (like SDL prerequisites), `kivy/pyjnius` has a limited set of dependencies.

And now, building `pyjnius` is definitely simpler than before.

So, I agree.